package.json

```diff
@@ -46,6 +46,7 @@
     "dev-server": "nodemon --watch ./tools/dev-server ./tools/dev-server/dev-server.js"
   },
   "dependencies": {
+    "ansis": "^2.0.3",
```

Comment:
I guess it's only required as a devDependency,  since I don't use it within the node helper, only in a development tool.

Reply:
Okay, moved it to devDependencies.